bin/make_db

```diff
@@ -1,5 +0,0 @@
-"""Initialize the dev environment's DB."""
-#!/usr/bin/env python3
-import sys
-from pyramid.paster import bootstrap
-bootstrap("conf/development.ini")
```

Comment:
This `bin/make_db` script didn't work in report (or other non-Pyramid projects): it contains Pyramid stuff. The cookiecutter has removed it